KeyShot/6.3.0/channels/root/content/rights_management/internal_access.dcl

```diff
@@ -1,14 +1,17 @@
 data channel_folder internal_access {
     name = 'Internal access'
     parent_id = data.channel_folder.rights_management.channel_folder_id
+    system = true
 }
 
 data item_security internal_access__internal_access {
     accessor_item_id = data.member_group.internal_access_29.item_id
     item_id = data.channel_folder.internal_access.item_id
+    system = true
 }
 
 patch item_security id_16061 {
     target = data.item_security.internal_access__internal_access
     write = true
+    system = true
```

Comment:
```suggestion
```
This is already set in the MM layer, so no need to patch it here as well 😉 
A patch works by updating the properties that are explicitly mentioned, so by leaving the `system` property out here, we preserve the value of `true` that is set in the MM layer.